src/circuit/num.rs

```diff
@@ -696,6 +696,48 @@ impl<E: Engine> AllocatedNum<E> {
         Ok(())
     }
 
+    /// Takes two allocated numbers (a, b) and returns
+    /// allocated boolean variable with value `true`
+    /// if the `a` smaller than `b`, `false` otherwise.
+    pub fn smaller<CS>(
```

Comment:
smaller => less_than (commonly abbreviated as lt)

Reply:
Also such functions is usually called “less than it equal” or “less” depending if it’s >= or >